iDiff/utils/image_prep_utils.go

```diff
@@ -115,11 +117,18 @@ func getHistory(imgPath string) ([]string, error) {
 
 func getImageFromTar(tarPath string) (string, error) {
 	glog.Info("Extracting image tar to obtain image file system")
-	err := ExtractTar(tarPath)
-	if err != nil {
-		return "", err
-	}
 	path := strings.TrimSuffix(tarPath, filepath.Ext(tarPath))
+	args := []string{"iDiff/undocker.py", "--tar", tarPath, "-o", path, "-v"}
+	undockerCmd := exec.Command("python", args...)
```

Comment:
I already have the code translated to Go in another branch that was intended to be merged after this PR went through.  I'm now working through trying to work it in to get the tests passing.  Also I see the undocker.py file in the "Files changed" Tab so I think it is added in the PR... (also see undocker.py here: https://github.com/abbytiz/runtimes-common/blob/e0e40a3e3829ce29df26e7ee6a6067cc0980d2c7/iDiff/undocker.py)